exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/EndToEndTest.java

```diff
@@ -89,33 +95,8 @@ public void exportMockSpanDataList(){
 
     spanDataList.add(spanDataOne);
 
-    List<Span> expectedSpans = new ArrayList<>();
-    Span spanOne = Span.newBuilder()
-            .setName("projects/" + PROJECT_ID + "/traces/" + TRACE_ID.toLowerBase16() + "/spans/" + SPAN_ID.toLowerBase16())
-            .setSpanId(SPAN_ID.toLowerBase16())
-            .setDisplayName(TruncatableString.newBuilder().setValue(SERVER_PREFIX + SPAN_NAME).setTruncatedByteCount(0).build())
-            .setStartTime(com.google.protobuf.Timestamp.newBuilder().setSeconds(START_SECONDS).setNanos(START_NANOS).build())
-            .setAttributes(ATTRIBUTES)
-            .setTimeEvents(TIME_EVENTS)
-            .setStatus(SPAN_STATUS)
-            .setEndTime(com.google.protobuf.Timestamp.newBuilder().setSeconds(END_SECONDS).setNanos(END_NANOS).build())
-            .setLinks(LINKS)
-            .setParentSpanId(PARENT_SPAN_ID.toLowerBase16())
-            .setSameProcessAsParentSpan(BoolValue.of(true))
-            .build();
-    expectedSpans.add(spanOne);
-
-    // Instead of doReturn, doAnswer is better to deal with whenever batchWriteSpans() is called.
-    // Rather than actually call the BatchWriteSpans() method from the API, null is returned.
-    doAnswer(invocation -> null)
-            .when(mockTraceServiceClient).batchWriteSpans(eq(PROJECT_ID), anyList());
-
     // Invokes export();
     assertEquals(SpanExporter.ResultCode.SUCCESS, exporter.export(spanDataList));
-
-    // Make sure that the parameters passed to batchWriteSpans() are what we expect them to be
-    verify(mockTraceServiceClient, times(1)).batchWriteSpans(eq(ProjectName.of(PROJECT_ID)),
```

Comment:
I removed this check since `verify` is a `Mockito` thing. This shouldn't be needed anymore since the export won't be successful if the span translation didn't work properly (the mock server has validations for these), and there are unit tests for `TraceTranslator` now

Reply:
Could you also delete https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/tree/master/exporters/trace/src/test/resources/mockito-extensions please?

Reply:
And also delete dependency on mockito in https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/6f0beb2bed141677fc37911fa4855120ec189673/exporters/trace/build.gradle#L11 and https://github.com/GoogleCloudPlatform/opentelemetry-operations-java/blob/6f0beb2bed141677fc37911fa4855120ec189673/build.gradle#L34